src/Microsoft.AspNetCore.WebUtilities/BufferedReadStream.cs

```diff
@@ -10,7 +10,11 @@
 
 namespace Microsoft.AspNetCore.WebUtilities
 {
-    internal class BufferedReadStream : Stream
+    /// <summary>
+    /// A Stream that wraps another stream and allows reading lines
```

Comment:
End the sentence with a period. Doc comments are kind of like HTML, where newlines are just whitespace. For that reason, every sentence needs to be a complete sentence because it will all show up as one paragraph in the final docs.